openwisp_ipam/templates/admin/openwisp-ipam/subnet/change_form.html

```diff
@@ -37,6 +46,12 @@ <h3>{% trans 'Subnet Visual Display' %}</h3>
 {{ block.super }}
 {% if original and not is_popup %}
 <script type="text/javascript">
+  document.querySelector("#goto-input").addEventListener("keyup", event => {
+    event.preventDefault();
+    if (event.key == "Enter") {
+      document.querySelector("#subnet-go").click();
+    }
+  });
```

Comment:
For some reason I can't get the input field to stop submitting the form when "enter" is clicked. I even tried to override it with the code above but no luck. I think this could be an issue with the admin theme from https://github.com/openwisp/openwisp-utils/. If yes, I would do the honors of opening an issue there :smile: